### PR TITLE
Add information about Mendix Cloud Dedicated

### PR DESCRIPTION
**Pull request by MarkvanMents**

### CODE REVIEW

File: content/en/docs/developerportal/deploy/_index.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ From within Mendix Studio Pro, you can view the cloud nodes you have available i
 
 For more information, see [Mendix Cloud](/developerportal/deploy/mendix-cloud-deploy/).
 
+### 3.1 Mendix Cloud Dedicated
+
+Large companies can also have their own Mendix Cloud, named [Mendix Cloud Dedicated](https://www.mendix.com/evaluation-guide/app-lifecycle/mendix-cloud-overview/#mendix-cloud-vpc). This works in exactly the same way as the Mendix Cloud, but runs on your own virtual private cloud and will only host apps your company.
```
Comment (feddovanede) [inline content/en/docs/developerportal/deploy/_index.md]: and will only host apps _of/by_  your company. ?

**Review by feddovanede — COMMENTED** [review]
1 small text update (I think).

Update to Support doc looks ok to me!

File: content/en/docs/developerportal/deploy/_index.md (hunk 1 above)
Comment (MarkvanMents) [reply content/en/docs/developerportal/deploy/_index.md]: Thanks. Yes, there is a word missing. I'll change it to "your company's apps"

**Comment by MarkvanMents** [comment]
Approved over email.

**Pull request merged**